Formula/python.rb

```diff
@@ -218,6 +218,19 @@ def install
         doc.install Dir["build/html/*"]
       end
     end
+
+    # Remove commands shadowing system python.
+    rm_rf bin/"easy_install"
+    rm_rf bin/"idle"
+    rm_rf bin/"pip"
+    rm_rf bin/"pydoc"
+    rm_rf bin/"python"
+    rm_rf bin/"python-config"
+    rm_rf bin/"pythonw"
+    rm_rf bin/"smtpd.py"
+    rm_rf bin/"wheel"
+    (libexec/"bin").mkpath
+    ln_sf bin/"python2", libexec/"bin"
```

Comment:
The file name in libexec/bin still ends up as `python2` not `python`. Also, I think we need unversioned symlinks in libexec/bin for the others, too (especially python-config and pip, but probably best to do all of them).